<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyWindow</class>
 <widget class="QDialog" name="ModifyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Correo</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>741</width>
     <height>163</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="timeLB">
          <property name="text">
           <string>Hora de envío:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeLE">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="idLB">
          <property name="text">
           <string>Identificador:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idLE">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="dateLB">
          <property name="text">
           <string>Fecha de Envío:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dateLE">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="fromLB">
        <property name="text">
         <string>Remitente:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fromLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="toLB">
        <property name="text">
         <string>Destinatario:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="toLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="ccLB">
        <property name="text">
         <string>Cc:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ccLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ccoLB">
        <property name="text">
         <string>Cco:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ccoLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="affairLB">
        <property name="text">
         <string>Asunto:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="affairLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>741</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPlainTextEdit" name="contentTE">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>168</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="backBT">
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyBT">
      <property name="text">
       <string>Aplicar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="contentLB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>739</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contenido:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
